recipes/plothist/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "plothist" %}
+{% set version = "1.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/plothist-{{ version }}.tar.gz
+  sha256: b4606d9ff7db3cc1a7f6f5d5440fd959e1d1292818910cc0d142b23e65e4de76
+
+build:
+  entry_points:
+    - install_latin_modern_fonts = plothist.scripts.install_latin_modern_fonts:install_latin_modern_fonts
+  noarch: python
```

Comment:
Failing on Windows for `noarch` Python projects is allowed given what I've been told by conda-forge core people in the past so this is fine.

Reply:
In this particular case the package won't be installable on Windows. If that is by design, we can merge this. However, it does seem like this package can be OS-noarch. See https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies for more details.

Reply:
@ocefpaf In that case, given https://github.com/conda-forge/staged-recipes/pull/27414#discussion_r1734616076 maybe we can wait for @0ctagon to make a patch release with https://github.com/cyrraz/plothist/pull/176 merged and then we can have that patch release be the one that generates the feedstock (so we'd update the version and sha256).

Reply:
B/c there is a patch already we can:

- ignore that, merge it, and you'all fix it as soon as there is a new release
- backport the patch now

I don't think it is worth making it OS-noarch b/c the fix is there already.

Reply:
> ignore that, merge it, and you'all fix it as soon as there is a new release

@ocefpaf I'm find with this suggestion. There should be a new patch release on PyPI before the end of the week so it would only be broken on Windows for a few days.